ts/src/okx.ts

```diff
@@ -5112,6 +5113,136 @@ export default class okx extends Exchange {
         return this.filterByArrayPositions (result, 'symbol', symbols, false);
     }
 
+    async fetchPositionsForSymbol (symbol, params = {}) {
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            // instType String No Instrument type, MARGIN, SWAP, FUTURES, OPTION
+            'instId': market['id'],
+            'instType': this.convertToInstrumentType (market['type']),
+            // posId String No Single position ID or multiple position IDs (no more than 20) separated with comma
+        };
+        // okx has one endpoint-method for all market-types
+        const response = await this.privateGetAccountPositions (this.extend (request, params));
```

Comment:
to say in a broad, across exchanges they might not return same, because endpoints that are used for "getting positions for single symbol" vs "getting positions of all symbols" have different endpoints across different exchanges. but on some exchanges they might have same endpoint.
in case of different endpoints, the response fields might also vary, and ofc rate-limits also vary. 
this new fPFS method is specially curated for single symbol position endpoints.

Reply:
@ttodua I'm aware of that, just wanted to confirm that in this case would not be different. 
With that said would it make sense to just return this.fetchPositions([symbol])` here?